src/Controls/src/Core/Handlers/Items/StructuredItemsViewHandler.Windows.cs

```diff
@@ -205,6 +210,9 @@ static ListViewBase CreateGridView(GridItemsLayout gridItemsLayout)
 				ItemContainerStyle = GetItemContainerStyle(gridItemsLayout)
 			};
 
+			//Hooked SizeChanged event for FormsGridView here, to unhook this event properly on DisconnectHandler.
+			gridView.SizeChanged += OnGridViewSizeChanged;
```

Comment:
Did you find the cause of why the SizeChanged event for the ItemsWrapGrid is not triggered when the CollectionView size is reduced? https://github.com/dotnet/maui/blob/48f19fd1435962119713ff1a8a87f05e0ad2c227/src/Controls/src/Core/Platform/Windows/CollectionView/FormsGridView.cs#L96

Reply:
@jsuarezruiz, This issue is replicated in native WinUI itself. The SizeChanged event of ItemsWrapGrid is not triggered properly when setting the item size in ItemsWrapGrid during resizing. Therefore, the SizeChanged event of GridView, which is the parent of ItemsWrapGrid, was used to dynamically adjust the ItemsWrapGrid dimensions.